content/api/servo-assembly/_index.md

```diff
@@ -7,6 +7,7 @@ The Servo Assembly is used to connect to multiple types of peripheral:
 - [GPIO pins](gpio) (digital and analogue)
 - [Servos](servos)
 - [Ultrasound sensors](ultrasound)
+- [Custom Arduino commands](arduino-commands)
```

Comment:
I'm not sure this link fits here, as this isnt a peripheral that's communicated with through the arduino.

Reply:
Good point; I'd not actually read the preceding line!

Reply:
9784c1a129d5388d2c8f2b4883835bf00b765484